virtual_rainforest/core/axes.py

```diff
@@ -164,6 +177,10 @@ def get_validator(axis: str, data: Data, darray: DataArray) -> Optional[Callable
     #   empty set.
     for (ld_dims, ld_coords, ld_grid_type), ld_fnm in AXIS_VALIDATORS[axis].items():
```

Comment:
What does `ld` stands for here? Maybe give a more descriptive name - or explain somewhere what that is?

Reply:
Hah - it was an abbreviation for `loaders` before I renamed lots of things. They should be `validator_dims` or some abbreviation of that.

Reply:
OK, so `loader` is the same as `validator`? I think I've seen used loader in other places, too, so it might be worth re-checking the naming.